app/services/update_stock_price.rb

```diff
@@ -0,0 +1,22 @@
+require 'iex-ruby-client'
```

Comment:
No need to add this `require 'iex-ruby-client'`

Reply:
It seems strange, as when I don't add `require `require 'iex-ruby-client'`, the code fails as it is looking for the module. So I added that to explicitly import it. Might there be a problem with my setup? Please enlighten me. TYIA.